core/trino-main/src/test/java/io/trino/security/TestFileBasedSystemAccessControl.java

```diff
@@ -819,23 +820,23 @@ public void testRefreshing()
     public void testAllowModeIsRequired()
     {
         assertThatThrownBy(() -> newAccessControlManager(createTestTransactionManager(), "catalog_allow_unset.json"))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessageStartingWith("Invalid JSON file");
+                .isInstanceOf(ProvisionException.class)
```

Comment:
Any reason for this change ?

Reply:
I changed a way how we get SystemAccessControl - it is fetched by injector now. See FileBasedSystemAccessControlFactory.create method. Previously it was created directly. IllegalArgumentException is now wrapped by ProvisionException. I added checking cause exception which is still IllegalArgumentException

Reply:
Do we need this change in this commit ?